includes/Products.php

```diff
@@ -543,7 +552,7 @@ public static function get_product_gender( \WC_Product $product ) {
 	 */
 	public static function update_product_gender( \WC_Product $product, $gender ) {
 
-		// TODO: implement
+		$product->update_meta_data( Products::GENDER_META_KEY, $gender );
```

Comment:
Same here as https://github.com/facebookincubator/facebook-for-woocommerce/pull/1482#pullrequestreview-473784482 - needs to save the meta 👍